### DISCUSSION TITLE
ASIO/CORE AUDIO SUPPORT

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#1176] [Ideas]
I am a music producer 
 i am recording all my instruments including vsti through my analogue console ( channel strip ).
If  audio gridder server provide an ASIO/CORE AUDIO support for a parallel analogue output of each active instrument channel,(selected/opened instruments in the plugin side) that will be very useful to all hardware users. In that scenario, we can control Instruments through AG plugin and can record through analogue channel strip.
now multiple audio is available via plugin over ip. but if we get an analogue stereo from the server side through ASIO/CORE AUDIO of an active instrument channel of DAW will be fine for analogue gear users.
may thanks 
Aby

**Comment by apohl79** [comment]
Hey Aby,

this sounds like #1159.

Cheers,
Andreas

**Reply by abysalvinthomas** [reply]
ys
